File: sample/java/com/example/java/BlockCommentTrim.java
package com.example.java;

import java.util.List;

public class BlockCommentTrim {
    /**
     * Foo is awesome.
     *
     * @return a number.
     */
    public int foo() {
        return 0; // asd
    }

    /**
     * @return a number.
     */
    public int  bar() {
        return 0;
    }

    /**
     * @param t     a.
     * @param items b.
     */
    public <T> void baz(T t, List<T> items) {}
}

File: sample/kotlin/com/example/kotlin/BlockCommentTrim.kt
package com.example.kotlin

class BlockCommentTrim {
    /**
     * Foo is awesome.
     *
     * @return a number.
     */
    fun foo(): Int = 0

    /** Foo is awesome. */
    fun bar(): Int = 0

    /**
     * @param t     a.
     * @param items b.
     */
    fun <T> baz(t: T, items: List<T>) {}
}
